w_titanic

```diff
@@ -0,0 +1,12 @@
+library(tidyr)
+library(dplyr)
+
+heart <- read.csv("titanic_original.csv", header = TRUE, stringsAsFactors = FALSE)
+
+heart$embarked[which(is.na(heart$embarked))] <- "S"
+heart$age[which(is.na(heart$age))] <- mean(heart$age, na.rm = TRUE)
+heart$boat[which(is.na(heart$boat))] <- "NA"
```

Comment:
I don't think heart.boat hash NA entries either; you can see this with:

table(is.na(dat$boat))

Instead, compare to empty strings.